.tekton/kserve-controller-213-pull-request.yaml

```diff
@@ -7,8 +7,11 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
-      == "rhoai-2.13"
+    pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request" 
+      && target_branch == "rhoai-2.13" 
+      && ( !".tekton/**".pathChanged() || ".tekton/kserve-controller-212-pull-request.yaml".pathChanged() ) 
```

Comment:
it should be tekton/kserve-controller-213-pull-request.yaml".pathChanged() ) on line 13